### PR TITLE
Add dynamic compose for dokuwiki

### PR DESCRIPTION
**Pull request by viktor-c**
Changed configs to dynamic compose for dokuwiki.
Checked on a fresh install.

## Summary by CodeRabbit

- **New Features**
	- Introduced dynamic configuration support for DokuWiki.
	- Added a new Docker Compose configuration for the DokuWiki service.

- **Updates**
	- Updated DokuWiki version references to `"2024-02-06b-ls250"`.
	- Modified Docker Compose image to use the latest DokuWiki image version.

These changes enhance configuration flexibility and ensure users benefit from the latest features and improvements.

### CODE REVIEW

**Comment by Lancelot-Enguerrand** [comment]
Hello @viktor-c,

Thank you for your contribution, this one is **greatly appreciated**. 😊

Can you make some modifications :
- Remove the addPorts section
- Update docker compose.yml to reflect the new dokuwiki version

File: apps/dokuwiki/docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,7 @@
-version: "3"
-
 services:
   dokuwiki:
     container_name: dokuwiki
-    image: lscr.io/linuxserver/dokuwiki:2022-07-31a-ls158
+    image: lscr.io/linuxserver/dokuwiki:latest
```
Comment (Lancelot-Enguerrand) [inline apps/dokuwiki/docker-compose.yml:4]: Change to same version as config.json

**Comment by viktor-c** [comment]
@Lancelot-Enguerrand  Thank you for your comments, changed the code as wished. 
I had originally changed to a fixed version because codeRabbit suggested above to do this
`"Consider pinning to a specific version instead of using 'latest'."`

**Review by Lancelot-Enguerrand — CHANGES_REQUESTED** [review]
Hello @viktor-c
I may have been unclear on the version to use, my bad.
We prefer using  fixed version for apps, to avoid unforseen breaking changes.

Which tag here ?
latest &rarr; no
version-2024-02-06b &rarr; almost equivalent to latest, so no
2024-02-06b-ls250 &rarr; stable, won't change, so yes

I hope to have cleared out any doubt.
Feel free to ask some questions if needed.

We should be ready to merge after this.

PS : CodeRabbit can be very useful but is subject to hallucination as it is an AI.
PS2 : It's not really a rabbit, sadly 😪

**Comment by viktor-c** [comment]
I updated my branch https://github.com/viktor-c/runtipi-appstore/tree/dokuwiki-dynamic-compose
but I don't see the latest update here with the pull request... Can it be because the PR is closed?

**Review by Lancelot-Enguerrand — APPROVED** [review]

**Pull request merged**